ghostjs-core/src/ghostjs.js

```diff
@@ -130,6 +130,7 @@ class Ghost {
    * Injects javascript and other things we need.
    */
   onOpen () {
+    console.log('on open called')
```

Comment:
Can we remove for the time being? I would like a way for more verbose logs, but I want to allow people to opt into it.